plasmapy/physics/transport.py

```diff
@@ -1382,27 +2506,29 @@ def _nondim_tec_ji_held(hall, Z, field_orientation):
 
     allowed_Z = [1, 2, 'arbitrary']
     Z_idx = _check_Z(allowed_Z, Z)
-    r = np.abs(Z * hall)
+    # fixing overflow errors when exponentiating r by making a float
+    # instead of an int
+    r = float(np.abs(Z * hall))
 
     def f_beta_par_e(Z):
-        numerator = Z**(5 / 3)
-        denominator = 0.693 * Z**(5 / 3) - 0.279 * Z**(4 / 3) + Z + 0.01
+        numerator = Z ** (5 / 3)
+        denominator = 0.693 * Z ** (5 / 3) - 0.279 * Z ** (4 / 3) + Z + 0.01
         return numerator / denominator
 
     def f_beta_0(Z):
-        return 0.156 * Z**(8 / 3) + 0.994 * Z**2 + 3.21 * Z - 0.84
+        return 0.156 * Z ** (8 / 3) + 0.994 * Z ** 2 + 3.21 * Z - 0.84
 
     def f_beta_1(Z):
-        return 3.69 * Z**(5 / 3) + 3.77 * Z + 0.77
+        return 3.69 * Z ** (5 / 3) + 3.77 * Z + 0.77
```

Comment:
For cases like this, I wonder if it would be helpful to use something like a dictionary with `0`, `1`, `2`, etc. as the keys and then a lambda function as the value, instead of having several separate functions.  This might be another case where we put a `# TODO: ` comment!

Reply:
Make this a new issue. The Braginskii stuff isn't code from my PR 😆